src/main/java/GeneralService/Configurations.java

```diff
@@ -0,0 +1,46 @@
+package GeneralService;
+
+import onlineservices.OnlineService;
+import onlineservices.services.WindowControl.WindowControlService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
+    private static final Logger LOGGER = LogManager.getLogger(Configurations.class.getName());
+    @Autowired
+    private WindowControlService windowControlService;
```

Comment:
delete this, private WindowControlService windowControlService;
you can get the object directly from windowControlService()